CodeTrans/docker_compose/amd/gpu/rocm/README.md

```diff
@@ -0,0 +1,123 @@
+###### Copyright (C) 2024 Advanced Micro Devices, Inc.
```

Comment:
This line will lead a doc IO build issue, suggest to remove it from docs.

Reply:
Deleted this line from the added files